subsys/bluetooth/host/smp.c

```diff
@@ -3009,6 +3035,8 @@ static int smp_send_security_req(struct bt_conn *conn)
 static uint8_t smp_pairing_req(struct bt_smp *smp, struct net_buf *buf)
 {
 	struct bt_conn *conn = smp->chan.chan.conn;
+	const struct bt_conn_auth_cb *bt_auth_cb = get_auth_cb(smp);
```

Comment:
Perhaps `bt_auth_cb` is a poor choice if `auth_cb` is the global pointer. I would expect `bt_auth_cb` to be the global one in that case, given the namespace. 

Suggest to rename to `smp_auth_cb`.

Reply:
ok, aligned.